src/Routes.js

```diff
@@ -9,12 +9,12 @@ import CoursePage from "./pages/CoursePageViewWithVideo/IndividualCourse";
 import MyCourses from "./pages/CoursePageViewWithVideo/index";
 import WishlistPage from "./pages/wishListView";
 import ProfilePage from "./pages/ProfilePage/index";
-import CourseDetail from "./pages/CourseDetail";
+import CourseDetail from "./pages/CourseDetail/index";
+import Checkout from "./pages/CheckoutView/index";
```

Comment:
Why this change?
It is not needed, The default file that is taken when we refer to any folder is index.js, so we can ignore that.

Also, I have added package-lock in .gitignore. So there will be no more packae-lock.json file changes if you create your next PR from the updated branch.

Also please make PR from seperate branches specially if you can omit main branch as your PR branch it will be very good, because github-rebase bot doesn't work on contributor's main branch.

Reply:
But it is showing error while running , that's why I had to add this index file.